lib/base-compiler.js

```diff
@@ -52,6 +52,12 @@ class BaseCompiler {
         if (!this.compiler.optArg) this.compiler.optArg = "";
         if (!this.compiler.supportsOptOutput) this.compiler.supportsOptOutput = false;
 
+        if (!this.compiler.disabledFilters)
```

Comment:
Thanks for this! I have to look deeper but I think we might have something with similar functionallity already

Reply:
I did look for it, but I could only find an ad-hoc explicit check for executable and binary (probably my bad, though :-) ). One of my problems is that I couldn't simply disable the buttons on the UI, because they start activated.